falcon/testing/base.py

```diff
@@ -26,14 +26,16 @@
 
 
 class TestBase(unittest.TestCase):
-    """Extends ``testtools.TestCase`` to support WSGI integration testing.
+    """Extends ``testtools.TestCase`` to support WSGI functional testing.
```

Comment:
I went ahead and reconciled the docstrings between the old and new classes. Let me know what you think!